src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -47,6 +47,8 @@ public class TermsOfUseHelper
         private readonly PackageManagerClient packageManagerClient;
         private readonly AuthenticationManager authenticationManager;
 
+        private static TermsOfUseView termsOfUseView;
```

Comment:
I am usually not a fan of keeping an instance/reference of view in view model since most of Dynamo code should follow MVVM model. Can you think of a different approach? For example, making the TermsOfUseView view a singleton?

Reply:
We can create a flag that checks if the view is created.

Reply:
Please give it a try